applications/GeoMechanicsApplication/custom_processes/calculate_total_motion_process.cpp

```diff
@@ -0,0 +1,57 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License: geo_mechanics_application/license.txt
+//
+//  Main authors: Aron Noordam,
+//
+#include "calculate_total_motion_process.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/model_part.h"
+#include "includes/node.h"
+#include "includes/variables.h"
+
+namespace Kratos
+{
+
+CalculateTotalMotionProcess::CalculateTotalMotionProcess(ModelPart& rModelPart, const Parameters& rParameters)
+    : Process(Flags()), mrModelPart{rModelPart}, mResultsVariableName{""}, mIncrementalVariableName{""}
```

Comment:
We don't need to initialize these. std::string will be initialized with `""` by default
```suggestion
    : Process(Flags()), mrModelPart{rModelPart}
```